src/styles/modal.css

```
@@ -82,7 +117,57 @@ body.modal-open {
   margin: 0 0 8px;
 }
 
-.detail {
+.detail-content {
   max-height: 430px;
   overflow-y: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  font-family: "Montserrat";
```

Stylelint (17.7.0)

[error] Expected no quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)